### ISSUE TITLE
source-loader produces invalid output with CSF

### ISSUE THREAD

**Issue by Hypnosphi** [storybookjs/storybook#8018]
It only happens if default export is a one-liner and includes `parameters` field without bracket spacing.

**To Reproduce**
Replace https://github.com/storybookjs/storybook/blob/next/examples/official-storybook/stories/addon-cssresources.stories.js#L3-L5 with following line:
```
export default {title: 'Addons|Cssresources', parameters: {foo: 'bar'}};
```

Error output:
```
WARNING in ./stories/addon-cssresources.stories.js
Module build failed (from /Users/jetbrains/IdeaProjects/storybook/node_modules/babel-loader/lib/index.js):
SyntaxError: /Users/jetbrains/IdeaProjects/storybook/examples/official-storybook/stories/addon-cssresources.stories.js: Unexpected token, expected "," (22:1973)

  20 | import React from 'react';
  21 | 
> 22 | export default {title: 'Addons|Cssresources', parameters: {"storySource":{"source":"import React from 'react';\n\nexport default {title: 'Addons|Cssresources', parameters: {foo: 'bar'}};\n\nexport const primaryLargeButton = () => (\n  <button type=\"button\" className=\"btn btn-lg btn-primary\">\n    Primary Large Button\n  </button>\n);\nprimaryLargeButton.story = {\n  name: 'Primary Large Button',\n  parameters: {\n    cssresources: [\n      {\n        id: `bootstrap v4.1.3`,\n        code: `<link rel=\"stylesheet\" type=\"text/css\" href=\"https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css\"></link>`,\n        picked: true,\n      },\n      {\n        id: `bootstrap v3.3.5`,\n        code: `<link rel=\"stylesheet\" type=\"text/css\" href=\"https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css\"></link>`,\n        picked: false,\n      },\n    ],\n    options: {\n      selectedPanel: 'storybook/cssresources/panel',\n    },\n  },\n};\n\nexport const cameraIcon = () => <i className=\"fa fa-camera-retro\"> Camera Icon</i>;\ncameraIcon.story = {\n  name: 'Camera Icon',\n  parameters: {\n    cssresources: [\n      {\n        id: `fontawesome`,\n        code: `<link rel=\"stylesheet\" type=\"text/css\" href=\"https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css\"></link>`,\n        picked: true,\n      },\n      {\n        id: `whitetheme`,\n        code: `<style>.fa { color: #fff }</style>`,\n        picked: false,\n      },\n    ],\n    options: {\n      selectedPanel: 'storybook/cssresources/panel',\n    },\n  },\n};\n","locationsMap":{"addons-cssresources--primary-large-button":{"startLoc":{"col":34,"line":5},"endLoc":{"col":1,"line":9},"startBody":{"col":34,"line":5},"endBody":{"col":1,"line":9}},"addons-cssresources--camera-icon":{"startLoc":{"col":26,"line":31},"endLoc":{"col":82,"line":31},"startBody":{"col":26,"line":31},"endBody":{"col":82,"line":31}}}},foo: 'bar'};
     |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      ^
  23 | 
  24 | export const primaryLargeButton = addSourceDecorator(() => (
  25 |   <button type="button" className="btn btn-lg btn-primary">

```

Looks like closing `}` is missing in the end.

**Workaround**
Add bracket spacing or line breaks:
```
export default { title: 'Addons|Cssresources', parameters: { foo: 'bar' } };
```
```
export default {
  title: 'Addons|Cssresources',
  parameters: {foo: 'bar'},
};
```

**Comment by shilman** [comment]
Cc @libetl

**Comment by libetl** [comment]
Ack

**Comment by libetl** [comment]
Hi, I am still here if you need help

**Comment by libetl** [comment]
Will take a look soon.

**Comment by vanessayuenn** [comment]
Hi! I am going through and cleaning up old issues in the repo. We just shipped a ton of improvements in Storybook 7.0, and I suspect this bug might be fixed already. Can you please try it out? If this issue persists with the latest version, please open a new bug report with a 7.x reproduction. Thank you so much! 🙏🏼
